src/main/java/com/rtb/projectmanagementtool/comment/CommentController.java

```diff
@@ -0,0 +1,120 @@
+package com.rtb.projectmanagementtool.comment;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query.*;
+import java.util.ArrayList;
+import java.util.List;
+
+/** Class controlling the CommentData object. */
+public final class CommentController {
+
+  private DatastoreService datastore;
+  private static final Filter NO_QUERY_FILTER = null;
+  private static final int NO_QUERY_LIMIT = Integer.MAX_VALUE;
+  private static final SortPredicate NO_QUERY_SORT = null;
+
+  public CommentController(DatastoreService datastore) {
+    this.datastore = datastore;
+  }
+
+  // Add methods
+
+  public void addComments(ArrayList<CommentData> comments) {
+    ArrayList<Entity> commentEntities = new ArrayList<>();
+    for (CommentData comment : comments) {
+      commentEntities.add(comment.toEntity());
+    }
+    addKeysToComments(comments, new ArrayList<>(datastore.put(commentEntities)));
+  }
+
+  private void addKeysToComments(ArrayList<CommentData> comments, ArrayList<Key> keys) {
+    if (comments.size() == keys.size()) {
+      for (int i = 0; i < keys.size(); i++) {
+        comments.get(i).setCommentID(keys.get(i).getId());
+      }
+    }
+  }
+
+  // Get methods
+
+  public CommentData getCommentByID(long commentID) {
```

Comment:
ooc what use cases would we return only a single comment?

Reply:
I'm not sure yet, but I figure it's useful to have.